docker/.env.example

```
@@ -0,0 +1,20 @@
+# User/Group IDs for file permissions
+# Set these to your user's UID/GID to avoid permission issues with mounted volumes
+MY_UID=1000
+MY_GID=1000
+
+# Profile Service Configuration
+# Path where the profile service stores its database
+DATABASE_PATH=/data/profiles.db
+
+# reverse proxy domain
+DOMAIN=staging.circlesubi.network
+SERVER_IP="135.181.238.49"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)